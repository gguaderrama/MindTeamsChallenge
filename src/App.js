import * as React from "react";
import {
  Routes,
  Route,
} from "react-router-dom";

import SignInSide from "./SignInSide";
import Dashboard from "./dashboard";


export default function App() {
  return (
    <div>
      <Routes>

          <Route path="/" element={<SignInSide/>} />
          <Route path="/dashboard" element={<Dashboard/>} />
          <Route path="/users" element={<Dashboard/>} />
          <Route path="/account" element={<Dashboard/>} />
          <Route path="/movements" element={<Dashboard/>} />

      </Routes>
    </div>
  );
}


