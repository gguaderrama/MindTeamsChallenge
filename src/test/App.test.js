import { render, screen } from '@testing-library/react';
import React from 'react';
import App from '../App';


import {BrowserRouter as Router} from 'react-router-dom';

describe('App tests', () => {
    it('should contains the login', () => {
      const container = render(<Router><App /></Router>);
      const element = container.queryByText('<h1 class="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root">Sign in</h1>');
      expect(element).not.toBeInTheDocument();
    });
});