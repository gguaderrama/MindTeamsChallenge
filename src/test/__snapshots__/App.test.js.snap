// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App tests should contains the login 1`] = `null`;
